__tests__/components/OrgPicker/OrgPicker.test.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @jest-environment jsdom
+ */
+import { describe, expect, it } from '@jest/globals';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { OrgPicker } from '@/components/OrgPicker/OrgPicker';
+
+describe('<OrgPicker />', () => {
+  describe('on initial load', () => {
+    it('content is collapsed', () => {
+      // act
+      render(<OrgPicker />);
+      // assert
+      const content = screen.queryByText('View all organizations');
+      expect(content).not.toBeInTheDocument();
+    });
+  });
+
+  describe('when button is clicked', () => {
+    it('content expands', async () => {
+      // setup
+      render(<OrgPicker />);
+      // act
+      const button = screen.getByAltText('open list');
+      fireEvent.click(button);
```

Comment:
I'm so glad you mentioned this because it forced me to try it again. I was having trouble, but then I followed [this guidance](https://testing-library.com/docs/user-event/intro#writing-tests-with-userevent) to use the result from `userEvent.setup()`. That worked for me! 
Note: I also used the `button` role instead of the image alt, since that's a more realistic interaction. Happy to discuss more in the ole Slack 😉

Reply:
love it, good call!